vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/scale/client.go

```diff
@@ -89,6 +98,14 @@ func (c *scaleClient) pathAndVersionFor(resource schema.GroupResource) (string,
 		return "", gvr, fmt.Errorf("unable to get full preferred group-version-resource for %s: %v", resource.String(), err)
 	}
 
+	// TODO(directxman12): this is a dirty, dirty hack because oapi just appears in discovery as "/v1", like
+	// the kube core API.  We can remove it if/when we get rid of the legacy oapi group entirely.  It makes me
+	// cry a bit inside, but such is life.
+	glog.Infof("DEBUG: %#v == %#v: %v", gvr, dcGVR, gvr == dcGVR)
```

Comment:
The hack is ok, but not the info level debug message.  Similar comment about making this a one liner with a `patch_dc.go` file.

Reply:
whoops, that must've slipped in, sorry.